packages/sync-service/lib/electric/postgres/replication_client.ex

```diff
@@ -359,7 +382,14 @@ defmodule Electric.Postgres.ReplicationClient do
         {:noreply, %{state | txn_collector: txn_collector}}
 
       {%Transaction{} = txn, txn_meta, %Collector{} = txn_collector} ->
-        state = %{state | txn_collector: txn_collector, last_seen_txn_lsn: txn.lsn}
+        timestamp = System.monotonic_time()
+
+        state = %{
+          state
+          | txn_collector: txn_collector,
+            last_seen_txn_lsn: txn.lsn,
+            last_seen_txn_timestamp: timestamp
```

Comment:
```suggestion
            last_seen_txn_timestamp: System.monotonic_time()
```
nit: seems clearer than binding a generic `timestamp` var